src/components/manifold-subscription-details/manifold-subscription-details.tsx

```diff
@@ -44,10 +54,50 @@ export class ManifoldSubscriptionCreate {
   }
 
   render() {
-    if (this.data?.subscription) {
-      return <h3>{this.data.subscription.plan.displayName}</h3>;
+    if (!this.data) {
+      return null;
     }
 
-    return null;
+    const { plan, status } = this.data.subscription;
+
+    return (
+      <div class="ManifoldSubscriptionCreate__Details">
+        {this.heading && <h1 class="ManifoldSubscriptionCreate__Heading">{this.heading}</h1>}
+        <section class="ManifoldSubscriptionCreate__Card">
+          <header class="ManifoldSubscriptionCreate__Details__Header">
+            <h2 class="ManifoldSubscriptionCreate__PlanName">{plan.displayName}</h2>
+            <h3 class="ManifoldSubscriptionCreate__SubscriptionStatus" data-status={status.label}>
+              <div class="ManifoldSubscriptionCreate__SubscriptionStatusIndicator" />
+              {status.label}
+            </h3>
+          </header>
+          <dl class="ManifoldSubscriptionCreate__Details__FeatureList">
+            {plan.fixedFeatures.edges.map(fixedFeature => (
+              <FixedFeature fixedFeature={fixedFeature as PlanFixedFeatureEdge} />
+            ))}
+            {plan.meteredFeatures.edges.map(meteredFeature => (
+              <MeteredFeature meteredFeature={meteredFeature as PlanMeteredFeatureEdge} />
+            ))}
+            {plan.configurableFeatures.edges.map(configurableFeature => (
+              // TODO format configured feature so it displays as fixed
+              <ConfigurableFeature
+                setConfiguredFeature={() => null}
+                configurableFeature={configurableFeature as PlanConfigurableFeatureEdge}
+              />
+            ))}
```

Comment:
These should be displays the same as fixed features probably. Maybe we can use fixed feature component and refactor the data into the right shape?

Reply:
Do you plan to do that in this PR, or start another one?

Reply:
probably another one